internal/gpsd/gpsd.go

```diff
@@ -235,3 +243,24 @@ func errUnexpected(err error) error {
 	}
 	return err
 }
+
+// GetGridSquare provides function for getting updated position and calculating the grid square.
+func (c *Conn) GetGridSquare() (string, error) {
```

Comment:
I believe `NextGridSquare()` would be a better name for this, to better harmonize with `Next()` and `NextPos()` 🙂

Reply:
Absolutely!